python/lsst/sims/photUtils/PhotometricParameters.py

```diff
@@ -1,7 +1,37 @@
 from builtins import object
-import numpy
+import numpy as np
+from lsst.sims.photUtils import Sed, Bandpass
+
+__all__ = ["PhotometricParameters", 'Dust_values']
+
+
+class Dust_values(object):
+    """Calculate extinction values
+    """
+    def __init__(self, R_v=3.1):
+        # Calculate dust extinction values
```

Comment:
Why bother with the minimum/maximum wavelength values? This is calculated once, presumably (if anyone's using this class reasonably), and so calculating over the entire wavelength range is not a big overhead, but avoids potential mistakes if our wavelength sensitivity shifts or someone is trying to use this for a new bandpass which doesn't match the dictionary.

Reply:
Ok .. so suggestion: 
Why not make the filterlist an argument to DustValues, with defaults of a tuple which is 
filterlist = ('u', 'g', 'r', 'i', 'z', 'y')
and then iterate through for filtername in filterlist, try to read the bandpass from disk. 
Sims_photUtils has a requirement to set up $throughputs, so this should be available (os.path.join(os.getenv($THROUGHPUTS_DEFAULT), 'total_' + filtername + '.dat')
.. if someone wants to add filters, they can point to a different directory with the environment variable and name their file appropriately, or we could even do something beyond that like allowing them to pass in their own filterpath. 

For dust in particular, this seems like it could be useful for JHK, etc, and broaden the options of use. (think, using MAF metrics to evaluate Euclid observations, etc.).